integration-tests/tests/workspace-dashboard.js

```diff
@@ -12,6 +12,13 @@ const testGoogleWorkspace = _.flow(
   await page.goto(testUrl)
   await viewWorkspaceDashboard(page, token, workspaceName)
   await findText(page, 'About the workspace')
+
+  // Check selected items in cloud information
+  const currentDate = new Date().toLocaleDateString()
+  await click(page, clickable({ text: 'Cloud information' }))
+  await findText(page, 'Cloud NameGoogle Cloud Platform')
+  await findText(page, `Bucket SizeUpdated on ${currentDate}0 B`)
```

Comment:
I don't have the Google project ID to look for that (as opposed to the Azure case where I have the resource group ID because the server response is mocked). I'm just checking a couple of things, including the bucket size info which is obtained through a separate Google call that we only do for Google workspaces.